datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/domain/CreateDomainResolver.java

```diff
@@ -45,9 +52,10 @@ public CompletableFuture<String> get(DataFetchingEnvironment environment) throws
 
     final QueryContext context = environment.getContext();
     final CreateDomainInput input = bindArgument(environment.getArgument("input"), CreateDomainInput.class);
+    final Urn parentDomain = input.getParentDomain() != null ? UrnUtils.getUrn(input.getParentDomain()) : null;
 
     return CompletableFuture.supplyAsync(() -> {
-
+      // todo - implement parentDomain permissions
```

Comment:
We don't need to do this todo, I'll push a PR to remove it

Reply:
Actually since it's on your branch, can you update it

Reply:
you bet